### PR TITLE
Disabled UPnP on local games

### PR DESCRIPTION
**Pull request by Mailaender**
Currently, UPnP isn't implemented in an asynchronous way so it has potential to block things. This disables it for skirmishes and missions. Restarting which involves a server disconnect and startup should now be faster.

### CODE REVIEW

**Review by abcdefg30 — APPROVED** [review]
Code changes lgtm.

**Comment by Dzierzan** [comment]
It would be great if it was reviewed by one more person. I require this change for my OpenSA mod.

File: OpenRA.Mods.Common/Effects/RallyPointIndicator.cs [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
 namespace OpenRA.Mods.Common.Effects
 {
-	class RallyPointIndicator : IEffect, IEffectAboveShroud, IEffectAnnotation
+	public class RallyPointIndicator : IEffect, IEffectAboveShroud, IEffectAnnotation
```
Comment (phrohdoh) [inline OpenRA.Mods.Common/Effects/RallyPointIndicator.cs:20]: How is this change related to disabling upnp on local games?

Comment (abcdefg30) [reply OpenRA.Mods.Common/Effects/RallyPointIndicator.cs:20]: Not at all, but the commit message explains: `Allow mod code to set rally point indicators.` I figured @Mailaender just wanted to get this easy fix in without needing to open another PR.

**Review by pchote — APPROVED** [review]

**Comment by pchote** [comment]
https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/d33215c2f862f2177826f22bb60e4dad05461091

File: OpenRA.Mods.Common/Effects/RallyPointIndicator.cs (hunk 1 above)
Comment (Mailaender) [reply OpenRA.Mods.Common/Effects/RallyPointIndicator.cs:20]: See https://github.com/OpenRA/OpenRA/pull/18673 for reference.
